cmake/Tools/Platform/Android/generate_android_project.py

```diff
@@ -410,7 +410,7 @@ def main(args):
                                                         gradle_version=gradle_version,
                                                         gradle_plugin_version=android_gradle_plugin_version,
                                                         override_ninja_path=override_ninja_path,
-                                                        include_assets_in_apk=parsed_args.get_argument(INCLUDE_APK_ASSETS_ARGUMENT_NAME),
+                                                        include_assets_in_apk=parsed_args.include_apk_assets,
```

Comment:
I'll update the documentation https://www.o3de.org/docs/user-guide/platforms/android/#setup-requirements